### PR TITLE
Added singly linked list find operation in C++

### PR DESCRIPTION
**Pull request by Dhenderson10**
**Fixes issue:**

None.

**Changes:**

Added an algorithm to return true if an element is found in a linked list, false otherwise.

### CODE REVIEW

File: code/data_structures/src/list/singly_linked_list/operations/find/find.cpp [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+// Part of Cosmos by OpenGenus Foundation
+struct Node
+{
+    Node *nextNode;
+    int data;
+
+    Node() //constructor
+    {
+        nextNode = NULL;
```
Comment (arnavb) [inline code/data_structures/src/list/singly_linked_list/operations/find/find.cpp]: Use `nullptr`.

File: code/data_structures/src/list/singly_linked_list/operations/find/find.cpp [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+        data = 0;
+    }
+};
+
+// Singly Linked List
+struct LinkedList
```
Comment (arnavb) [inline code/data_structures/src/list/singly_linked_list/operations/find/find.cpp]: Why not use a class with some member functions?

File: code/data_structures/src/list/singly_linked_list/operations/find/find.cpp [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+    }
+}
+
+// Return true if an element is found within a linked list
+// otherwise return false
+bool find(LinkedList *list, int value)
+{
+    Node *currentNode = new Node;
+    currentNode = list->head;
+    while (currentNode != NULL)
```
Comment (arnavb) [inline code/data_structures/src/list/singly_linked_list/operations/find/find.cpp]: while(currentNode) also works.

File: code/data_structures/src/list/singly_linked_list/operations/find/find.cpp (hunk 2 above)
Comment (Dhenderson10) [reply code/data_structures/src/list/singly_linked_list/operations/find/find.cpp]: Just preference, doesn't matter to me.

**Pull request closed without merging**